src/main/java/com/pusher/rest/Pusher.java

```diff
@@ -3,13 +3,13 @@
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.net.URI;
-import java.util.Collections;
-import java.util.List;
-import java.util.ArrayList;
-import java.util.Map;
+import java.nio.charset.StandardCharsets;
+import java.util.*;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
```

Comment:
I don't think  this is necessary anymore with `import java.util.*;`.

Reply:
asterisks aren't recursive :(